test_programs/plonky2_trace/2_function_calls/expected_trace.json

```diff
@@ -0,0 +1,22 @@
+[
+  { "Path": "" },
+  { "Function": { "path_id": 0, "line": -1, "name": "<toplevel>" } },
+  { "Call": { "function_id": 0, "args": [] } },
+  { "Type": { "kind": 30, "lang_type": "None", "specific_info": { "kind": "None" } } },
+  { "Path": "?" },
```

Comment:
What "?" means here?

Reply:
Good catch; SourceLocation::create_unknown() creates { path: ?, line: -1 }, which is where this "?" comes from. But I should be using it for <toplevel> as well.

TODO in a future commit.